rpa-mapper/src/test/java/uk/gov/hmcts/cmc/rpa/mapper/DefenceResponseJsonMapperTest.java

```diff
@@ -0,0 +1,162 @@
+package uk.gov.hmcts.cmc.rpa.mapper;
+
+import org.json.JSONException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.response.DefenceType;
+import uk.gov.hmcts.cmc.domain.models.response.YesNoOption;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleAddress;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleParty;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleResponse;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleTheirDetails;
+import uk.gov.hmcts.cmc.domain.utils.ResourceReader;
+import uk.gov.hmcts.cmc.rpa.config.ModuleConfiguration;
+
+import java.time.LocalDateTime;
+
+import static org.skyscreamer.jsonassert.JSONAssert.assertEquals;
+import static org.skyscreamer.jsonassert.JSONCompareMode.STRICT;
+
+@SpringBootTest
+@ContextConfiguration(classes = ModuleConfiguration.class)
+@RunWith(SpringJUnit4ClassRunner.class)
+@SuppressWarnings({"LineLength"})
+public class DefenceResponseJsonMapperTest {
+
+    private static String INDIVIDUAL = "/defence/defence_response_individual_rpa_case.json";
```

Comment:
why do the file names have defence and response in them?
individual_rpa_case.json is fine as it is namespaced under defence

you could probably drop it down to just `individual.json` as it is namespaced under rpa-mapper/response

Reply:
There're going to be loads of JSON due to testing in coming days. I've left RPA text so that when searched on IDE, it become bit intuitive. I understand namespace argument of yours but I'm looking at from day-to-day programming on IDE.